app/components/job/details/job-progress-status/job-progress-status.component.spec.ts

```diff
@@ -124,4 +131,18 @@ describe("JobProgressStatusComponent", () => {
         const el = de.query(By.css(".completed .failed"));
         expect(el.nativeElement.textContent).toContain("2");
     });
+
+    it("doesn't show warning if data is validated", () => {
+        const warning =  de.query(By.css(".invalidated-data"));
+        expect(warning).toBeFalsy();
+    });
+
+    it("show warning about task count validity if count is unvalidated", () => {
```

Comment:
It doesn't, we talked about this at lunch, the problem also is how do we know if there is more than 200k tasks if it can't count it.

The idea was maybe to disable the gauge if it was over 200k

Reply:
I'll append to the warning if total count is close to 200k

Reply:
Try to choose a phrase from the feature article.   The feature has a very complicated consistency model.

Reply:
We can chat in person as well... Ideally the text would match what we surface in the portal.